src/lib/dfp/display-ads.ts

```diff
@@ -21,8 +22,14 @@ const displayAds = (): void => {
 	window.googletag.pubads().collapseEmptyDivs();
 	window.googletag.enableServices();
 
-	loadAdvert(dfpEnv.advertsToLoad[0]);
-	dfpEnv.advertsToLoad.length = 0;
+	const firstAdvertToLoad: Advert | undefined = dfpEnv.advertsToLoad.length
+		? dfpEnv.advertsToLoad[0]
+		: undefined;
+
+	if (firstAdvertToLoad) {
+		loadAdvert(firstAdvertToLoad);
+		dfpEnv.advertsToLoad.length = 0;
```

Comment:
Might not be the best place to raise this, but it's driving me a bit loopy. Can we update to:

```suggestion
		dfpEnv.advertsToLoad = [];
```

As far as I can tell, [we don't keep references to the original array](https://github.com/search?q=repo%3Aguardian%2Fcommercial%20dfpEnv.advertsToLoad&type=code) anywhere (which would be horrible to keep track of) and in [other](https://github.com/guardian/commercial/blob/de2bb5039bf670bc123560a613c7a57e6bc4870f/src/lib/dfp/lazy-load.ts#L33) [places](https://github.com/guardian/commercial/blob/de2bb5039bf670bc123560a613c7a57e6bc4870f/src/lib/dfp/display-lazy-ads.ts#L18) we totally create new arrays anyway.

Originally implemented in [frontend 13289](https://github.com/guardian/frontend/blame/4e9da3bde887ff0762bc43f117e55de4eac0e7b3/static/src/javascripts/projects/common/modules/commercial/dfp/dfp-api.js#L304)

Reply:
I think a valid suggestion @SiAdcock , and perhaps makes more logical sense looking at it. 
I would suggest amending it in this PR?

Reply:
Okay, go for it 👍 I'll take responsibility.

Thanks @dskamiotis